docs/tutorials/01_introductory/viz_earth_animation.py

```diff
@@ -0,0 +1,163 @@
+"""
+===============
+Texture Sphere Animation
+===============
+In this tutorial, we will show how to animate a textured sphere.
+"""
+
+import numpy as np
+from fury import window, actor, utils, primitive, io
+import itertools
+from fury.data.fetcher import read_viz_textures, fetch_viz_textures
+from fury.data.fetcher import read_viz_models, fetch_viz_models
+
+##############################################################################
+# Create a scene to start.
+
+scene = window.Scene()
+
+##############################################################################
+# Next, load in a texture for each of the actors. For this tutorial, we will
+# be creating one textured sphere for the Earth, and another for the moon.
+# Collect the Earth texture from the FURY github using ``fetch_viz_textures``
+# and  ``read_viz_textures``, then use ``io.load_image`` to load in the
+# image.
+
+fetch_viz_textures()
+filename = read_viz_textures("1_earth_8k.jpg")
+image = io.load_image(filename)
+
+##############################################################################
+# Using ``actor.texture_on_sphere()``, create an earth_actor with your newly
+# loaded texture.
+
+earth_actor = actor.texture_on_sphere(image)
+
+##############################################################################
+# Then, do the same for the moon.
+
+filename = read_viz_textures("moon-8k.jpg")
+image = io.load_image(filename)
+
+moon_actor = actor.texture_on_sphere(image)
+
+##############################################################################
+# Add both actors to the already existing scene.
+
+scene.add(earth_actor)
+scene.add(moon_actor)
+
+##############################################################################
+# Next, alter the position and scale of the moon to correctly size it in
+# comparison to the Earth using ``actor.SetPosition()`` and
+# ``actor.SetScale()``, and rotate the Earth using ``utils.rotate`` to
+# correctly align the texture.
+
+moon_actor.SetPosition(1, 0.1, 0.5)
+moon_actor.SetScale(0.25, 0.25, 0.25)
+utils.rotate(earth_actor, (-90, 1, 0, 0))
+
+##############################################################################
+# The ShowManager class is the interface between the scene, the window and the
+# interactor.
+
+showm = window.ShowManager(scene,
+                           size=(900, 768), reset_camera=False,
+                           order_transparent=True)
+
+##############################################################################
+# Next, let's focus on creating the animation.
+# We can determine the duration of animation with using the ``counter``.
+# Use itertools to avoid global variables.
+
+counter = itertools.count()
+
+##############################################################################
+# The timer will call this user defined callback every 200 milliseconds. The
+# application will exit after the callback has been called 200 times.
```

Comment:
this text does not fit with the code below (`set_camera`) can you:
- move this text somewhere else
- Add a new explanation for `set_camera`